crawl-ref/source/dat/des/branches/shoals.des

```diff
@@ -2110,6 +2110,53 @@ cww......L
 ccccJJKKLL
 ENDMAP
 
+NAME:   dreamdust_from_the_deep
+TAGS:   no_monster_gen no_item_gen no_tide
+DEPTH:  Shoals, !Shoals:$
+ORIENT: northwest
+WEIGHT: 3
+KFEAT:  X = open_sea
+SUBST:  # = xxx~
+NSUBST: ~ = 8:z / 2:Z / *:w
+KFEAT:  wzZ = deep_water
+NSUBST: - = 6:1 / 2:2 / *:-
+KFEAT:  -12 = shallow_water
+TILE:   -12 = dngn_shallow_water_murky
+: set_feature_name("shallow_water", "some foul and slimy water")
+FTILE:  .3G?x = floor_slime
+TILE:   x = wall_oozing
+TILE:   G = dngn_gravestone
+: set_feature_name("granite_statue", "a stone stele inscribed with bizarre and terrifying symbols")
+KMONS:  z = octopode zombie
+KMONS:  Z = kraken zombie
+KMONS:  1 = octopode ; trident ego:chaos | dagger ego:chaos
```

Comment:
We avoid using player species monsters that are not in normal circulation, mostly because they're defined for non-gameplay cosmetic reasons and aren't designed as interesting monsters in combat. In this case, I don't think there's a good connection to the Octopus King unrands, since this is an abyss-themed vault. I'd recommend simply using a Shoals-appropriate monster like a merfolk impaler and remove the placement of that unrand. The malign gateway spellbook and existing monster set more than establish the theme. 

I don't think the kraken zombie adds much to this vault either, other than being a somewhat weak tentacle connection. But there is no need for an additional undead theme. In general for thematic vaults, to get the point across I recommend choosing one strong thematic idea and developing that one idea well rather than using multiple semi-related themes. If you really like the Kraken, a live one would actually work perfectly fine. A Kraken zombie is also mostly strictly weaker than a normal Kraken. If you really like the "undead" connection, a spectral kraken would be more effective.